libs/containers/Guild.lua

```diff
@@ -424,6 +424,25 @@ function Guild:pruneMembers(days)
 	end
 end
 
+--[=[
+@m getBan
+@p user User-ID-Resolvable | User
+@r Ban
+@d Returns a [ban object](https://discordapp.com/developers/docs/resources/guild#ban-object) for the given user.
```

Comment:
Do not link to the Discord docs in our docstrings. A Discordia object is not the same as a Discord API JSON object. Also the link can break without warning.

Reply:
Also say something about how the object may not exist.